docs/api.md

```diff
@@ -12,6 +12,7 @@ const db = new Kinto(options);
 - `headers`: The default headers to pass for every HTTP request performed to the Cliquet server (eg. `{"Authorization": "Basic bWF0Og=="}`);
 - `adapter`: The persistence layer adapter to use for saving data locally (default: `Kinto.adapters.IDB`); alternatively, a `Kinto.adapters.LocalStorage` adapter is also provided; last, if you plan on writing your own adapter, you can read more about how to do so in the [Extending Kinto.js](extending.md) section.
 - `requestMode`: The HTTP [CORS](https://fetch.spec.whatwg.org/#concept-request-mode) mode. Default: `cors`.
+- `prefixDB`: The prefix for the local DB name (default: `""`). Use this option so that your application can switch remote server and authorization without combining records.
```

Comment:
Also, the sentence is hard to understand from a user perspective. Maybe *Use this option to isolate user specific databases*?